src/components/hero/_hero.scss

```diff
@@ -87,6 +87,12 @@
         z-index: 5;
     }
 
+    &--grey & {
+        &__details {
+            padding-top: 1rem;
+        }
+    }
```

Comment:
This needs to be changed when the breadcrumbs is used not when we use the grey variant